CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Grafana Mimir
 
+* [ENHANCEMENT] Ruler: Add `<prometheus-http-prefix>/api/v1/status/buildinfo` endpoint. #2724
```

Comment:
Please rebase, there are more entries in `main / unreleased` section now.

Reply:
rebased 👍